contracts/extensions/ColonyExtension.sol

```diff
@@ -42,7 +42,7 @@ abstract contract ColonyExtension is DSAuth {
   function deprecate(bool _deprecated) public virtual;
   function uninstall() public virtual;
 
-  function getDeprecated() public returns (bool deprecated) {
+  function getDeprecated() public view returns (bool) {
```

Comment:
The fact that we had to remove calling the return variable `deprecated` here took me a long time to find. Just highlighting it as something we fell in to and should keep an eye out to avoid in the future. Really, the takeaway here is to always test our functions, no matter how simple they look :smile: